### ISSUE TITLE
Create GitHub page for Flixel

### ISSUE THREAD

**Issue by IQAndreas** [FlixelCommunity/flixel#88]
The page should definitely host the ASDoc, allowing you to view the documentation from a browser, but perhaps it can contain other things such as download links to the current FlixelCommunity version of the SWC etc.

**Comment by Dovyski** [comment]
We still have some bugs to fix, but the list is getting smaller, so I think it's time to start working on this task. A page for FlixelCommunity is a great step further to organize and promote the project.

Does anyone of you guys have special interest on taking the lead on this one? If you don't, I can start working on something.

**Comment by moly** [comment]
I have no experience creating webpages, so feel free to start this yourself.

**Comment by IQAndreas** [comment]
I believe GitHub pages supports [markdown syntax](http://github.github.com/github-flavored-markdown/) (the same as is used in GitHub comments). Otherwise, Jekyll is very easy to set up and use. I think a simple default template should suffice for now.

**Comment by Dovyski** [comment]
Yeah, I agree. I will use a default page or a very simple Bootstrap one. No sub-pages or things like that, just some info and a download button.

**Comment by yadurajiv** [comment]
:+1:
